web_m2x_options/static/src/js/form.js

```diff
@@ -0,0 +1,393 @@
+                    search_val.length > 0 && !_.contains(raw_result, search_val) &&
+                    show_create) {
+                    values.push({
+                        label: _.str.sprintf(_t('Create "<strong>%s</strong>"'),
+                            $('<span />').text(search_val).html()),
+                        action: self._quickCreate.bind(self, search_val),
+                        classname: 'o_m2o_dropdown_option'
+                    });
+                }
+                // create and edit ...
+
+                var create_edit = self.is_option_set(self.nodeOptions.create) || self.is_option_set(self.nodeOptions.create_edit),
+                    create_edit_undef = _.isUndefined(self.nodeOptions.create) && _.isUndefined(self.nodeOptions.create_edit),
+                    m2x_create_edit_undef = _.isUndefined(self.ir_options['web_m2x_options.create_edit']),
+                    m2x_create_edit = self.is_option_set(self.ir_options['web_m2x_options.create_edit']);
+                var show_create_edit = (!self.nodeOptions && (m2x_create_edit_undef || m2x_create_edit)) || (self.nodeOptions && (create_edit || (create_edit_undef && (m2x_create_edit_undef || m2x_create_edit))));
+                if (create_enabled && !self.nodeOptions.no_create_edit && show_create_edit) {
+                    var createAndEditAction = function () {
+                        // Clear the value in case the user clicks on discard
+                        self.$('input').val('');
+                        return self._searchCreatePopup("form", false, self._createContext(search_val));
+                    };
+                    values.push({
+                        label: _t("Create and Edit..."),
+                        action: createAndEditAction,
+                        classname: 'o_m2o_dropdown_option',
+                    });
+                } else if (values.length === 0) {
+                    values.push({
+                        label: _t("No results to show..."),
+                    });
+                }
+                // Check if colors specified to wait for RPC
+                if (!(self.field_color && self.colors)){
+                    def.resolve(values);
+                }
+            });
+
+            return def;
+        }
+    });
+
+    FormFieldMany2ManyTags.include({
+        events: _.extend({}, FormFieldMany2ManyTags.prototype.events, {
+            'click .badge': '_onOpenBadge',
+        }),
+
+        start: function() {
+            this._super.apply(this, arguments);
+            return this.get_options();
+        },
+
+        get_options: function() {
+            var self = this;
+            if (_.isUndefined(this.ir_options_loaded)) {
+                this.ir_options_loaded = $.Deferred();
+                this.ir_options = {};
+                self._rpc({
+                    model: "ir.config_parameter",
+                    method: 'search_read',
```

Comment:
This model now only is readable by admins, so other users cannot get the configs. Just enter runbot and use a form as demo user.

You need to create a different method in the backend that reads the specified values with `.sudo()`. This can make the client code easier too.